src/index.html

```diff
@@ -7,8 +7,63 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link 
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header_logo" href="#0">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+
+      <nav class="header_nav">
+        <ul class="header_navList">
+          <a
+            href="#1"
+            class="header_navLink is-active"
+          >
+            apple
+          </a>
+          <a 
```

Comment:
Add empty line between multiline siblings (here you will have `<li>`)
```suggestion
          </a>
          
          <a 
```